documentation/prototype-usage-instructions.md

```diff
@@ -63,51 +98,322 @@ context of the document. This can be done with additional keys that
+   corresponding to the elements defined by MathML.
+
+LuaLaTeX (via the `luamml` package which is loaded automatically when
+needed) includes a basic TeX to MathML convertor so can generate both
+these forms. If using other engines then only the Associated File
+mechanism may be used.
+
+
+
+### Associated MathML Files
+By default, when using LuaLaTeX and `unicode-math` the mathematics will be
+tagged with an Associated File of the generated MathML.
 
-For this the document should be compiled once with the command
-```latex
-\tagpdfsetup{math/mathml/write-dummy}
 ```
+\DocumentMetadata{uncompress,lang=en,
+ testphase={phase-III,math,table,title},
+ pdfversion=2.0,pdfstandard=ua-2,pdfstandard=a-4f}
+
+\documentclass{article}
+\usepackage{unicode-math}
+
+\title{Math tagged with Associated Files (luamml)}
+\author{LaTeX Team}
+\begin{document}
+
+\maketitle
+
+\section{Basic mathematical expressions}
+
+If $x$ is real, then $x^{2} \geq 0$.
+
+A matrix equation.
+\[
+\begin{pmatrix}0&1\\1&0\end{pmatrix}
+\begin{pmatrix}a&b\\c&d\end{pmatrix}
+=
+\begin{pmatrix}c&d\\a&b\end{pmatrix}
+\]
+
+\end{document}
+```
+
+### MathML Structure Element Tagging
+
+To use MathML Structure Element tagging you may use the
+`math/mathml/structelem` key as shown below. Here we also suppress
+adding the Associated Files.
 
-This will write a file `<file>-mathml-dummy.html` which has a
-prepared section for every equation found in the document. The section
-shows the LaTeX-source and a hash value and an empty
-`<math></math>` tag.
-```HTML
-<div>
-<h2>\mml 1</h2>
-<p>$x=1$</p>
-<p>8FD5BB0EEAA8887F6A312C99359A3B93</p>
-<math></math>
-</div>
 ```
+\DocumentMetadata{uncompress,lang=en,
+ testphase={phase-III,math,table,title},
+ pdfversion=2.0,pdfstandard=ua-2,pdfstandard=a-4f}
+
+\tagpdfsetup{
+ math/mathml/structelem,
+ math/tex/AF=false,
+ math/mathml/AF=false
+ }
+
+\documentclass{article}
+\usepackage{unicode-math}
+
+\title{Math tagged with MathML Structure Elements}
+\author{LaTeX Team}
+\begin{document}
+
+\maketitle
+
+\section{Basic mathematical expressions}
+
+If $x$ is real, then $x^{2} \geq 0$.
+
+A matrix equation.
+\[
+\begin{pmatrix}0&1\\1&0\end{pmatrix}
+\begin{pmatrix}a&b\\c&d\end{pmatrix}
+=
+\begin{pmatrix}c&d\\a&b\end{pmatrix}
+\]
+
+\end{document}
+```
+
+### Supplying MathML to be used for Associated File tagging,
+
+For engines other than LuaTeX, or if you want to modify the automatcally
+generated MathML,then LaTeX needs to be supplied MathML in a separate file.
+
+The format of the file may be seen in the example below.  which has a
+prepared section for every equation found in the document. The section
+shows the LaTeX-source and a hash value and a `<math></math>` tag.
+LaTeX can write out a "dummy" version of this file with an empty math
+element for each formula which must be filled in before LaTeX is
+re-run to incorporate the MathML. This is controlled by use of the key
+`math/mathml/write-dummy`.
```

Comment:
maybe add  some commented out lines for this in the sample doc, so that people can (at least by copy-paste) try it easily on their local machine?

Reply:
99b0cc1